src/ui/feedmanagement/FeedManagementComponent.java

```diff
@@ -32,39 +32,43 @@ public VBox initialise() {
 		Node node = createListView();
 		node.setOnKeyReleased(((e) -> {
 			if (e.getCode().equals(KeyCode.ENTER)) {
-				TurboFeed selectedIssue = listView.getSelectionModel().selectedItemProperty().get();
-				if (selectedIssue != null) {
-					ui.triggerEvent(new IssueSelectedEvent(selectedIssue.getIssueNum(), -1));
-				}
+				showIssue();
 			}
 		}));
 		node.setOnMouseClicked(((e) -> {
-				TurboFeed selectedIssue = listView.getSelectionModel().selectedItemProperty().get();
-				if (selectedIssue != null) {
-					ui.triggerEvent(new IssueSelectedEvent(selectedIssue.getIssueNum(), -1));
-				}
+			showIssue();
 		}));
 		layout.getChildren().addAll(node);
 		VBox.setVgrow(listView, Priority.ALWAYS);
 		return layout;
 	}
-	
+
+	private void showIssue() {
+		TurboFeed selectedIssue = listView.getSelectionModel()
+				.selectedItemProperty().get();
+		if (selectedIssue != null) {
+			ui.triggerEvent(new IssueSelectedEvent(selectedIssue.getIssueNum(),
+					-1));
```

Comment:
`-1` shouldn't be used for the column index here. It *will* break a number of things. Since we're moving the feed into a column it shouldn't be that big a problem, but it should be done as soon as we can.